src/core/qgsofflineediting.cpp

```diff
@@ -383,7 +385,7 @@ void QgsOfflineEditing::initializeSpatialMetadata( sqlite3 *sqlite_handle )
   spatial_ref_sys_init( sqlite_handle, 0 );
 }
 
-bool QgsOfflineEditing::createSpatialiteDB( const QString &offlineDbPath )
+bool QgsOfflineEditing::createOfflineDb( const QString &offlineDbPath, bool gpkg )
```

Comment:
rather than a bool, I would use an enum, this would be more futureproof and flexible.